### PR TITLE
create project structure

### PR DESCRIPTION
**Pull request by xuanwgit**
1. Create a project structure

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# vs code test command
```
Comment (hgao62) [inline main.py]: please remove this test comment

**Review by hgao62 — COMMENTED** [review]
looks good

File: main.py (hunk 1 above)
Comment (plantepro) [reply main.py]: yes already removed

**Review by xuanwgit — COMMENTED** [review]
Already removed

**Pull request merged**